website/docs/concepts/html/introduction.mdx

```diff
@@ -114,25 +114,29 @@ html! {
   </TabItem>
   <TabItem value="SVG" label="SVG">
 
+:::note
+SVG tags currently require opting out of static typing by using dynamic elements.
```

Comment:
Why is this the case?

Reply:
See the PR description. It's the first point mentioned under limitation